.github/workflows/issues_and_pull_requests.yml

```diff
@@ -52,6 +52,8 @@ jobs:
           days-before-pr-close: "${{ env.DAYS_BEFORE_PR_CLOSE }}"
           days-before-pr-stale: "${{ env.DAYS_BEFORE_PR_STALE }}"
           exempt-all-milestones: true
+          exempt-issue-labels: "topic/core"
```

Comment:
maybe even some label like "prevent-stale" to prevent stable bot on non-core topics

Reply:
No, I want meaningful labels that serve real purpose of cataloguing stuff, so I prefer not to add label just for ignoring - it's better to use multiple explicit labels for each Stale Bot's option if needed.